app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
So I've been able to test it again, it's working well. But could you remove the `:page_ancher` parameter from the `service` state?